source/elm-live.js

```diff
@@ -183,10 +183,27 @@ ${indent(String(elmMake.error), 2)}
     unlinkDir: 'removed',
   };
 
+  // wait until the returned funtion has not been called for `wait`
+  // milliseconds before calling the passed in function
+  const debounce = (func, wait) => {
+    let timeout;
+    return (...theArgs) => {
+      const context = this;
+      const later = () => {
+        func.apply(context, theArgs);
```

Comment:
Any reason it’s not just `func(...theArgs)`? Since we use ES6 anyway, let’s just go for it! 😀